### PR TITLE
Receiver tab update for 4.3

### PR DESCRIPTION
**Pull request by ctzsnooze**
Incorporates #2667 and #2635 into a single PR - they are now closed.  All comments from those PR's have been addressed I think.

Changes the Receiver drop-down labels as follows:

|    Old   |   New   |
|----|----|
| PPM RX input | PPM/CPPM (single wire) |
| Serial-based receiver (SPEKSAT, SBUS, SUMD | Serial (Via UART) |
| PWM RX input (one wire per channel) | PWM (one wire per channel) |
| MSP RX input (control via MSP port) | MSP (control via MSP port) |
| SPI RX support | SPI (e.g. built-in Rx) |

Updates the general message at the top of the Receiver tab to emphasise the warnings - especially the ADC warning, which is really important with the newer, fast Rx links. Direct link from Failsafe note to our Failsafe Testing doc.  Slightly changed the serial message also.

Rearranges the html so that the the craft icon is in the left column, to make more effective use of the available space. Happy for the craft icon to go below the bars, but I couldn't figure how to do that, and it looks cool at the top I think.

Any other suggestions / ideas to improve the receiver tab for 4.3?

How it looks with this PR:

suggestedRxLayout

vs how it used to look:

prevRxLayout

### CODE REVIEW

**Review by limonspb — DISMISSED** [review]

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by KarateBrot — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
